crates/codegen/parser/runtime/src/napi_interface/cst.rs

```diff
@@ -54,7 +54,7 @@ impl RuleNode {
         self.0
             .children
             .iter()
-            .map(|child| child.to_js(&env))
+            .map(|child| child.to_js(env))
```

Comment:
I wonder if we can now stop using `JsObject` in return types, and return our types directly? since we override it anyways with `ts_return_type`.